README.md

```diff
@@ -7,7 +7,7 @@ The available commands are:
 
 `flatten`: transform a directory of parquet files with a nested structure into a directory of parquet files with a flat schema that can be loaded into impala or hive (neither of which support nested schemas)
```

Comment:
One more nit: Add a description of the default output to the paragraphs on each of flatten/tsv/compact?